src/App.tsx

```diff
@@ -15,47 +16,126 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+// Use this function in the render method to prepare goods
+export function getReorderedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  // To avoid the original array mutation
+  const visibleGoods = [...goods];
+
+  // Sort and reverse goods if needed
+  // eslint-disable-next-line no-console
+  console.log(sortType, isReversed);
+
+  if (sortType) {
+    visibleGoods.sort((g1, g2) => {
```

Comment:
not really good idea to name it like `g1` or `g2` or whatever. Names should be clear, even in the callbacks